neurots/generate/section.py

```diff
@@ -97,6 +97,9 @@ def _propose(self, extra_randomness=0, add_random_component=True):
 
         if add_random_component:
             random_component = self.params.randomness * get_random_point(random_generator=self._rng)
+            # this is needed only to get 100% invariance under y_direction
+            # if self.context.get("y_rotation") is not None:
+            #    random_component = self.context["y_rotation"].dot(random_component)
```

Comment:
Why is it commented?

Reply:
I see. Maybe it would be nice to make the comment above a bit more intelligible then.

Reply:
its difficult to make it invariant under noise, and not useful, so I removed this